launcher-env-template.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+#################################################
+# Here you can find basic setup for this script #
+#################################################
+
+# Choose (uncomment) one of the 3 KeyCloak options below
+KEYCLOAK=NO
+#KEYCLOAK=OFFICIAL
+#KEYCLOAK=LOCAL
+
+#################################################
+#################################################
+
+
+SCRIPT_DIR=$(cd "$(dirname "$BASH_SOURCE")" ; pwd -P)
+
+#### DO NOT CHANGE - Reset environment variables
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_TOKEN
+unset LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME
+unset LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN
+unset LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME
+unset LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN
+unset LAUNCHER_MISSIONCONTROL_BITBUCKET_USERNAME
+unset LAUNCHER_MISSIONCONTROL_BITBUCKET_APPLICATION_PASSWORD
+unset LAUNCHER_KEYCLOAK_URL
+unset LAUNCHER_KEYCLOAK_REALM
+unset LAUNCHER_MISSIONCONTROL_OPENSHIFT_CLUSTERS_FILE
+####
+
+case "$KEYCLOAK" in
+"NO")
+    # No KeyCloak
+    MSHIFT=$(minishift console --url)
+    if [[ $MSHIFT == *"://:"* ]]; then
+        echo "WARNING: MiniShift is NOT running, the environment variables will NOT be properly set!"
+    fi
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL=$MSHIFT
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=$MSHIFT
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME=developer
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=developer
+    # If set, will override username/password authentication scheme
+        #export LAUNCHER_MISSIONCONTROL_OPENSHIFT_TOKEN=<token here>
+
+    # Setup git providers default credentials (please look at README.md)
+    export LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=`git config github.user`
+    export LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=`git config github.token`
+    export LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME=`git config gitlab.user`
+    export LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN=`git config gitlab.token`
+    export LAUNCHER_MISSIONCONTROL_BITBUCKET_USERNAME=`git config bitbucket.user`
+    export LAUNCHER_MISSIONCONTROL_BITBUCKET_APPLICATION_PASSWORD=`git config bitbucket.password`
+    ;;
+"OFFICIAL")
+    # Official KeyCloak
+    export LAUNCHER_KEYCLOAK_URL=https://sso.openshift.io/auth
+    export LAUNCHER_KEYCLOAK_REALM=rh-developers-launch
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_CLUSTERS_FILE=$SCRIPT_DIR/clusters.yaml
+    ;;
+"LOCAL")
+    # Local KeyCloak
+    export LAUNCHER_KEYCLOAK_URL=http://localhost:8280/auth
+    export LAUNCHER_KEYCLOAK_REALM=launch
+    export LAUNCHER_MISSIONCONTROL_OPENSHIFT_CLUSTERS_FILE=$SCRIPT_DIR/clusters.yaml
```

Comment:
I think we need to explain this. Which probably means we need a KeyCloak section in the README.